lib/aullar/spec/styling_spec.moon

```diff
@@ -253,6 +253,24 @@ describe 'Styling', ->
           6, 's1:s2', 7
         }, styling\get(1, 7)
 
+        assert.same nil, styling\get_nearest_style_marker 1
```

Comment:
Don't throw in the new stuff in the middle of an existing spec, but create specific specs for the styling marker stuff instead. These existing specs deal with styling, and not with markers.